File: backend/apartment1/src/main/java/com/apartment/entity/Employee.java
package com.apartment.entity;
import jakarta.persistence.*;
import java.util.Date;

@Entity
@Table(name = "Employee")
public class Employee {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    @Column(length = 100, nullable = false)
    private String name;

    private Date birthDate;

    @Column(length = 100)
    private String email;

    @Column(length = 20)
    private String phoneNumber;

    @Column(length = 255)
    private String avatar;

    @Column(length = 100, nullable = false)
    private String username;

    @Column(length = 100, nullable = false)
    private String password;

    @ManyToOne
    @JoinColumn(name = "role_id", referencedColumnName = "id")
    private Roles role;

    private Boolean status;

    // Getters and Setters
}

File: backend/PHP/MockProkject-app/app/Models/Employee.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Employee extends Model
{
    // Khai báo tên bảng rõ ràng
    protected $table = 'employee';

    // Các cột có thể gán giá trị hàng loạt
    protected $fillable = [
        'name',
        'birth_date',
        'email',
        'phone_number',
        'avatar',
        'username',
        'password',
        'role_id',
        'status'
    ];

    // Nếu bảng không có các cột timestamps
    public $timestamps = false;
}
